docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG SUBSTRA_VERSION=latest
+FROM substra:$SUBSTRA_VERSION
```

Comment:
I would start from python and install from pip if it was me so that you don't depend on an intermediary image. Also you could use /usr/src/app as workdir as it would not be taken already.